docs/scenarios/actions.rst

```diff
@@ -42,33 +42,47 @@ Here is the syntax of the regexp action:
-matching the received message or header. MANDATORY. search_in msg can
-have four values: "msg" (try to match against the entire message);
-"hdr" (try to match against a specific SIP header); "body" (try to
-match against the SIP message body); or "var" (try to match against a
-SIPp string variable). header None Header to try to match against.
-Only used when the search_in tag is set to hdr. MANDATORY IF search_in
-is equal to hdr. variable None Variable to try to match against. Only
-used when the search_in tag is set to var. MANDATORY IF search_in is
-equal to var. case_indep false To look for a header ignoring case .
-Only used when the search_in tag is set to hdr. occurence 1 To find
-the nth occurence of a header. Only used when the search_in tag is set
-to hdr. start_line false To look only at start of line. Only used when
-the search_in tag is set to hdr. check_it false if set to true, the
-call is marked as failed if the regexp doesn't match. Can not be
-combined with check_it_inverse. check_it_inverse false Inverse of
-check_it. iff set to true, the call is marked as failed if the regexp
-does match. Can not be combined with check_it. assign_to None contain
-the variable id (integer) or a list of variable id which will be used
-to store the result(s) of the matching process between the regexp and
-the message. Those variables can be re-used at a later time either by
-using '[$n]' in the scenario to inject the value of the variable in
-the messages or by using the content of the variables for conditional
-branching. The first variable in the variable list of assign_to
-contains the entire regular expression matching. The following
-variables contain the sub-expressions matching. Example:
 
+================  ======= ===========
+Keyword           Default Description 
+================  ======= ===========
+regexp            None    Contains the regexp to use for
+                          matching the received message or header. MANDATORY. 
+search_in         msg     can have four values: "msg" (try to match against the entire message),
+                          "hdr" (try to match against a specific SIP header), "body" (try to
+                          match against the SIP message body), or "var" (try to match against a
+                          SIPp string variable). 
+header            None    Header to try to match against.
+                          Only used when the search_in tag is set to hdr. MANDATORY IF search_in
+                          is equal to hdr. 
+variable          None    Variable to try to match against. Only
+                          used when the search_in tag is set to var. MANDATORY IF search_in is
+                          equal to var. 
+case_indep        false   To look for a header ignoring case .
+                          Only used when the search_in tag is set to hdr. 
+occurence     1           To find the nth occurence of a header. Only used when the search_in tag is set
```

Comment:
Has been fixed to `occurrence` (double-r). Althrough single-r is still legal for backwards compatibility.